packages/firestore/src/model/target_index_matcher.ts

```diff
@@ -145,16 +146,22 @@ export class TargetIndexMatcher {
       return true;
     }
 
-    if (this.inequalityFilter !== undefined) {
+    if (this.inequalityFilters.size > 0) {
+      if (this.inequalityFilters.size > 1) {
+        // Only single inequality is supported for now.
+        return false;
+      }
+
+      // Only a single inequality is currently supported. Get the only entry in the map.
```

Comment:
nit: update the comment above: `Get the only entry in the set.`

Reply:
oops.